application/src/main/java/org/togetherjava/tjbot/commands/help/OnGuildBanDeleteSpamThreadListener.java

```diff
@@ -0,0 +1,43 @@
+package org.togetherjava.tjbot.commands.help;
+
+import net.dv8tion.jda.api.entities.User;
+import net.dv8tion.jda.api.events.guild.GuildBanEvent;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+import org.jooq.Result;
+
+import org.togetherjava.tjbot.commands.EventReceiver;
+import org.togetherjava.tjbot.db.Database;
+import org.togetherjava.tjbot.db.generated.tables.records.HelpThreadsRecord;
+
+import java.time.Duration;
+import java.time.Instant;
+
+import static org.togetherjava.tjbot.db.generated.Tables.HELP_THREADS;
+
+public final class OnGuildBanDeleteSpamThreadListener extends ListenerAdapter
```

Comment:
I don't like this name

SpamThreadDeletionListener?

Well first of all it shouldn't start with `On`, it's not a method

It should also be shorter

Reply:
another class in the same package have same naming

Reply:
I see, well I don't like it personally, find it *way too* specific, if JDA renames the method the class's name is obsolete already.

Reply:
its not supposed to match JDAs method signature. its just english text for us. "when someone leaves a guild, we delete threads".

that said, i dont like the following:
* its about scam, not spam
* why even mention that aspect. it deletes all recently created threads and thats okay
* the class name reads bad, "OnGuildBan", better would be "OnUserBanned"

so im voting for `OnUserBannedDeleteRecentThreadsListener`

Reply:
I really am not a fan of the "on" pattern, find it pretty disgusting tbh

Reply:
done